src/neo/SmartContract/Native/TrimmedBlock.cs

```diff
@@ -57,6 +57,22 @@ public void Serialize(BinaryWriter writer)
             writer.Write(Hashes);
         }
 
+        IInteroperable IInteroperable.Clone()
+        {
+            return new TrimmedBlock
+            {
+                Header = Header,
+                Hashes = Hashes
+            };
+        }
+
+        void IInteroperable.FromReplica(IInteroperable replica)
+        {
+            TrimmedBlock from = (TrimmedBlock)replica;
```

Comment:
```suggestion
            if (replica is not TrimmedBlock from) throw new ArgumentException(null, nameof(replica));
```

Reply:
If you don't throw an exception, it will also automatically throw the exception on conversion.